tests/test_appids.py

```diff
@@ -110,7 +110,28 @@ def test_celery() -> None:
     assert "celery: /path/to/app3 (/path/to/app3.py)" == get_application_name(
         process_with_cmdline(["celery", "a", "b", "--app=/path/to/app3"])
     )
-    # No app
+
+
+def test_celery_with_queue() -> None:
```

Comment:
Also test with multiple queues? How does it look like? `-Q a,b,c` or `-Q a -Q b`?

Reply:
`-Q a,b,c`